package.json

```diff
@@ -99,28 +95,27 @@
     "publint": "^0.2.4",
     "react": "18.3.1",
     "react-dom": "18.3.1",
-    "rimraf": "^5.0.5",
+    "rimraf": "6.0.1",
     "statuses": "^1.4.0",
     "tree-kill": "^1.2.2",
     "ts-jest": "^29.0.3",
     "tsup": "^8.0.1",
-    "turbo": "^2.0.14",
-    "turbo-ignore": "^2.0.6",
+    "turbo": "2.2.3",
```

Comment:
🙃 Do we want to update this in this PR? Just want to make sure we're isolating changes.

Reply:
Reverted to original `turbo` package versions